packages/sync/src/class-replicastore.php

```diff
@@ -878,12 +878,18 @@ public function get_terms( $taxonomy ) {
 	 *
 	 * @access public
 	 *
-	 * @param string $taxonomy   Taxonomy slug.
-	 * @param int    $term_id    ID of the term.
-	 * @param bool   $is_term_id Whether this is a `term_id` or a `term_taxonomy_id`.
+	 * @param string      $taxonomy   Taxonomy slug.
+	 * @param int         $term_id    ID of the term.
+	 * @param bool|string $is_term_id Whether this is a `term_id`, otherwise pass 'term_taxonomy_id'.
```

Comment:
Since false isn't supposed to be passed, maybe this would be clearer
```suggestion
	 * @param bool|string $is_term_id Pass `true` if this is a `term_id`, otherwise pass `'term_taxonomy_id'`.
```
Even better might be to forget the `true` part and always pass an appropriate string. Are there any callers actually passing true?

If we make it a string, we might also rename the parameter to not sound like a boolean.

The corresponding doc in Replicastore_Interface could probably also use an update. We might even rename the variable.